scripts/gitlab/build_and_test.sh

```diff
@@ -227,33 +227,12 @@ then
     # in case we want to make them disctinct in the future.
     #
 
-    if echo ${sys_type} | grep -q "blueos" && echo ${spec} | grep -q "cuda" ; then
-        if grep -q -i "CMAKE_BUILD_TYPE.*Release" ${hostconfig_path}
-        then
-            echo "~~~~~~~~~ Run Command: ~~~~~~~~~~~~~~~~~~~~~"
-            echo "lrun -n1 ... ctest --output-on-failure -T test"
-            echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-            lrun -n1 --smpiargs="-disable_gpu_hooks" ctest --output-on-failure -T test
-        else
-            echo "~~~~~~~~~ Run Command: ~~~~~~~~~~~~~~~~~~~~~"
-            echo "lrun -n1 ... ctest --output-on-failure -T test"
-            echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-            lrun -n1 --smpiargs="-disable_gpu_hooks" ctest --output-on-failure -T test
-        fi
-    else
-        if grep -q -i "CMAKE_BUILD_TYPE.*Release" ${hostconfig_path}
-        then
-            echo "~~~~~~~~~ Run Command: ~~~~~~~~~~~~~~~~~~~~~"
-            echo "ctest --output-on-failure -T test 2>&1 | tee tests_output.txt"
-            echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-            ctest --output-on-failure -T test 2>&1 | tee tests_output.txt
-        else
-            echo "~~~~~~~~~ Run Command: ~~~~~~~~~~~~~~~~~~~~~"
-            echo "ctest --output-on-failure -T test 2>&1 | tee tests_output.txt"
-            echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-            ctest --output-on-failure -T test 2>&1 | tee tests_output.txt
-        fi
-    fi
+    echo "~~~~~~~~~ Run Command: ~~~~~~~~~~~~~~~~~~~~~"
+    echo "ctest --output-on-failure -T test 2>&1 | tee tests_output.txt"
+    echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+    date
+    ctest --output-on-failure -T test 2>&1 | tee tests_output.txt
+    date
```

Comment:
@MrBurmark I don’t know what `--smpiargs="-disable_gpu_hooks"` is used for. Tests are running without it. Attempts to define it in lalloc failed.

Reply:
That is something you need on lassen if you're not running through lrun/mpi.